README.md

```diff
@@ -66,6 +69,15 @@ Additional details are provided below. Note that several hosts and/or guests are
 
 A later stage of SAMPL7 is expected to include logD prediction (hopefully with pKa values provided) for a series of small moleculs in several solvents; data is currently being collected in partnership with GSK.
 
+
+### Physical property challenges on pK<sub>a</sub>, partitioning, and permeability
+
+We have a new set of SAMPL7 challenges focusing on pK<sub>a</sub>, partitioning, and permeability. [The Ballatore group at UCSD](https://pharmacy.ucsd.edu/faculty/ballatore) is contributing a set of measured water-octanol log *P*, log *D*, and pK<sub>a</sub> values for 22 compounds. They also provide PAMPA permeability values they measured.
```

Comment:
I'd call it "a new SAMPL7 challenge", and make sure it's "(optionally) permeability".

Reply:
Updated